### PR TITLE
Issue 1: User can authenticate

### PR DESCRIPTION
**Pull request by solomonjin**
reddit authentication

### CODE REVIEW

File: client/components/mypaper.jsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* eslint-disable comma-dangle */
```
Comment (thebearingedge) [inline client/components/mypaper.jsx:1]: You can turn this off in `.eslintrc` if you really want to 😂

File: server/index.js [hunk 2]
```diff
@@ -2,11 +2,115 @@ require('dotenv/config');
 const express = require('express');
 const errorMiddleware = require('./error-middleware');
 const staticMiddleware = require('./static-middleware');
+const jwt = require('jsonwebtoken');
+const cookieParser = require('cookie-parser');
+const pg = require('pg');
+const snoowrap = require('snoowrap');
+const snoostorm = require('snoostorm');
+const ClientError = require('./client-error');
+// eslint-disable-next-line no-unused-vars
```
Comment (thebearingedge) [inline server/index.js]: It's not a good habit to disable rules like this. Just delete the unused variable until you need it.

File: server/public/_redirects [hunk 3]
```diff
@@ -0,0 +1 @@
+/* /index.html 200
```
Comment (thebearingedge) [inline server/public/_redirects]: What is this?

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: server/public/_redirects (hunk 3 above)
Comment (solomonjin) [reply server/public/_redirects]: Ah I forgot to remove that file. It was one of the many things I was trying to get browser routing to work properly on refresh.

File: client/components/mypaper.jsx (hunk 1 above)
Comment (solomonjin) [reply client/components/mypaper.jsx:1]: For some reason, the linter was only freaking out about the commas in that one module. If I had dangling commas, the linter would say it shouldn't be there, but when I got rid of it, the linter would tell me something else and would put in that comma automatically when I saved.
Screen Shot 2021-07-22 at 3 29 38 PM
Screen Shot 2021-07-22 at 3 29 45 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**